sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -876,6 +876,16 @@ def validate(self, validator):
 
     return errors
 
+  def lookup_dataflow_service_option(self, key, default=None):
+    if not self.dataflow_service_options:
+      return None
+    elif key in self.dataflow_service_options:
+      return default
+    for service_name in self.dataflow_service_options:
```

Comment:
`option_name` is would be a better name for this varilable. The service is Dataflow in this case.